pr_agent/tools/pr_reviewer.py

```diff
@@ -107,6 +107,17 @@ async def run(self) -> None:
             relevant_configs = {'pr_reviewer': dict(get_settings().pr_reviewer),
                                 'config': dict(get_settings().config)}
             get_logger().debug("Relevant configs", artifacts=relevant_configs)
+
+            if self.incremental.is_incremental and hasattr(self.git_provider, "file_set") and not self.git_provider.file_set:
```

Comment:
what happens there isnt "file_set" ?

please validate on gitlab and bitbucket that we give reasonable output